### PR TITLE
Add a codeql status bar item

### PR DESCRIPTION
**Pull request by aeisenberg**
Includes the current cli version as well as the canary status (codeQL.canary) in the settings.

## Checklist

- [x] [CHANGELOG.md](https://github.com/github/vscode-codeql/blob/main/extensions/ql-vscode/CHANGELOG.md) has been updated to incorporate all user visible changes made by this pull request.
- [n/a] Issues have been created for any UI or other user-facing changes made by this pull request.
- [n/a] `@github/docs-content-dsp` has been cc'd in all issues for UI or other user-facing changes made by this pull request.

### CODE REVIEW

**Comment by aeisenberg** [comment]
Squirrel icon, not a bird.

**Comment by aeisenberg** [comment]
Removing the squirrel. Too small and hard to see.

File: extensions/ql-vscode/src/status-bar.ts [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  }
+
+  private async updateStatusItem() {
+    const canary = CANARY_FEATURES.getValue() ? ' (Canary)' : '';
+    // since getting the verison may take a few seconds, initialize with some
+    // meaningful text.
+    this.item.text = `CodeQL${canary}`;
+
+    const version = await this.cli.getVersion();
+    this.item.text = `CodeQL cli v${version}${canary}`;
```
Comment (adityasharad) [inline extensions/ql-vscode/src/status-bar.ts:39]: Do you think we should include both the extension version and the CLI version here? (Is there room?)

```suggestion
    this.item.text = `CodeQL CLI v${version}${canary}`;
```

**Review by adityasharad — COMMENTED** [review]
Haven't had the chance to test it yet, but changes look good.

File: extensions/ql-vscode/src/status-bar.ts (hunk 1 above)
Comment (aeisenberg) [reply extensions/ql-vscode/src/status-bar.ts:39]: There's already a standard place to check the extension version. So, I'd rather not take up more precious space in the status bar.

**Review by adityasharad — APPROVED** [review]

**Pull request merged**